src/components/Footer/BottomBar.stories.tsx

```diff
@@ -11,9 +11,9 @@ type Story = StoryObj<typeof BottomBarComponent>;
 
 export const BottomBar: Story = {
   args: {
-    title: 'Accessibility Statement',
     email: 'team@example.com',
     onClickAccessibilityStatement: () => console.log('Click Accessibility'),
-    url: 'http://localhost:6006'
+    url: '/#accessibility/',
```

Comment:
I think it might be neater if the end slash '/' wasn't included

Reply:
url value would be empty in dashboard app due to use of routes so include everything url should be for drt app